cdap-ui/app/cdap/components/CaskMarketPlace/index.js

```diff
@@ -13,13 +13,36 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
-import React from 'react';
+
+import React, {Component} from 'react';
 
 import TabConfig from './TabConfig';
 import ConfigurableTab from '../ConfigurableTab';
+import {MyCaskMarketApi} from '../../api/caskmarket';
+import MarketAction from './action/market-action.js';
+import _ from 'lodash';
```

Comment:
One minor thing. We could do `import find from 'lodash/find'`. This way when we update to webpack 3+ then we can rollup to include only parts of lodash instead of the entire library.